test/library/packages/Sort/performance/performance.good

```diff
@@ -6,3 +6,5 @@ insertionSort (seconds): x.xxx
 binaryInsertionSort (seconds): x.xxx
 mergeSort (seconds): x.xxx
 selectionSort (seconds): x.xxx
+timSort (seconds): x.xxx
+
```

Comment:
This added newline is causing a failure in testing

Reply:
Thank you for review
UPD: fixed